.air.toml

```diff
@@ -4,8 +4,8 @@ tmp_dir = "tmp"
 
 [build]
   args_bin = ["serve"]
-  bin = "./tmp/envoy"
-  cmd = "go build -o ./tmp/envoy ./cmd/envoy"
+  bin = "./tmp/bin/envoy"
+  cmd = "go build -o ./tmp/bin/envoy ./cmd/envoy"
```

Comment:
@daniellemaxwell hopefully this will be merged seamlessly and you won't notice any problems; but I wanted to call out this change to the `.air.toml` file -- if you're having trouble with `air` let me know.